### PR TITLE
[DOCS] Updated community.md

### PR DESCRIPTION
**Pull request by developer-guy**
I added great Katacoda tutorial for OpenFaas made by Jonathan Johnson

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Postponed events for 2020
 | [Applying GitOps to OpenFaaS with Flux Helm Operator](https://www.openfaas.com/blog/openfaas-flux/) | Stefan Prodan | openfaas.com| 16-Jan-2020| 
 | [OpenFaaS 3rd Birthday Celebrations](https://www.openfaas.com/blog/birthday-teamserverless/) | Alex Ellis | openfaas.com| 14-Jan-2020| 
 | [How To Get Started Contributing to OpenFaaS](https://blog.heyal.co.uk/developing-openfaas/)| Alistair Hey | blog.heyal.co.uk| 04-Jan-2020 |
-
+| [OpenFaaS on Kubernetes](https://www.katacoda.com/courses/serverless/openfaas)| Jonathan Johnson |www.katacoda.com/courses/serverless| 04-Jan-2020 |
```
Comment (alexellis) [inline community.md:132]: Change the URL to "katacoda.com" please

**Comment by alexellis** [comment]
Hi, the date is actually 22 Sep 2018 (looking at the GitHub repo - https://github.com/javajon/katacoda-courses/blob/master/kubernetes-serverless/openfaas/assets/nodeinfo.yaml)

**Comment by developer-guy** [comment]
> Hi, the date is actually 22 Sep 2018 (looking at the GitHub repo - https://github.com/javajon/katacoda-courses/blob/master/kubernetes-serverless/openfaas/assets/nodeinfo.yaml)

I fixed the section.

**Comment by developer-guy** [comment]
> Hi, the date is actually 22 Sep 2018 (looking at the GitHub repo - https://github.com/javajon/katacoda-courses/blob/master/kubernetes-serverless/openfaas/assets/nodeinfo.yaml)
> 
> 
> 
> 

Updated thank you .

**Comment by alexellis** [comment]
Hi, I know this is your first PR, so I've merged the change manually to compress your three separate changes into one. You'll see the file in the history and should show up as a contributor now. 

Alex

**Comment by developer-guy** [comment]
> Hi, I know this is your first PR, so I've merged the change manually to compress your three separate changes into one. You'll see the file in the history and should show up as a contributor now. 
> 
> 
> 
> Alex

Thank you 🙏

**Pull request closed without merging**